### PR TITLE
Added README and move script to "tools" directory

### PR DESCRIPTION
**Pull request by yhqs540**

### CODE REVIEW

**Comment by ryanmcdowell** [comment]
Overrode CLA verification after confirming both git users have signed the CLA.

File: tools/cloudera-parcel-gcsconnector/README.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
+# Cloudera parcel installation
+
+This script helps you create a Cloudera parcel that includes Google Cloud Storage connector. The parcel can be deployed on a Cloudera managed cluster.
+
+# Prerequisites
```
Comment (ryanmcdowell) [inline tools/cloudera-parcel-gcsconnector/README.md]: Should be sub-heading

```md
## Prerequisites
```

File: tools/cloudera-parcel-gcsconnector/README.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+1. Download create_parcel.sh script file.
+2. [Optional] If you want the script to deploy the parcel file under Cloudera Manager's parcel repo directly, you need to use the script on the Cloudera Manager server.
+3. [Optional] For on-premise clusters or clusters on other cloud providers, you need to get the service account JSON key from your GCP account. You can follow steps from [this document](https://cloud.google.com/iam/docs/creating-managing-service-account-keys).
+
+# Installing
```
Comment (ryanmcdowell) [inline tools/cloudera-parcel-gcsconnector/README.md]: Should be sub-heading

```md
## Installation
```

File: tools/cloudera-parcel-gcsconnector/README.md [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+-d is to be used if you want to deploy the parcel to the cloudera manager parcel repo folder, this flag is optional and if not provided then the parcel file will be created in the same directory where create_parcel.sh is script run.
+
+Example
+------
+We can name this parcel as “gcsconnector”, version as “1.0.0”, and os type rhel6, run the below command.
+```
+$ ./create_parcel.sh -f gcsconnector -v 1.0.0 -o el6 -d
+```
+
+# Deployment
```
Comment (ryanmcdowell) [inline tools/cloudera-parcel-gcsconnector/README.md]: Should be sub-heading

```md
## Deployment
```

File: tools/cloudera-parcel-gcsconnector/README.md [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+You can check the parcel by go to the Cloudera Manager Home page, click the **Hosts** > **Parcels** > **Check parcels**. Once the new parcel populates in the list of parcels.
+Click **Distribute** > **Activate parcel**. This will distribute and activate the parcel on all Cloudera managed hosts.
+
+Once activated successfully, **restart** all the stale services.
+
+Check below path for logs:
+/var/log/build_script.log
+
+
+# Configure CDH services to use GCS connector
```
Comment (ryanmcdowell) [inline tools/cloudera-parcel-gcsconnector/README.md]: Should be sub-heading

```md
## Configure CDH services to use GCS connector
```

File: tools/cloudera-parcel-gcsconnector/README.md [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+
+# HDFS service
```
Comment (ryanmcdowell) [inline tools/cloudera-parcel-gcsconnector/README.md]: Should be sub-heading

```md
### HDFS service
```

File: tools/cloudera-parcel-gcsconnector/README.md [hunk 6]
```diff
@@ -0,0 +1,107 @@
...
+
+**fs.gs.project.ids** : GCP project ID
+
+**fs.gs.impl** : com.google.cloud.hadoop.fs.gcs.GoogleHadoopFileSystem
+
+![alt text](screenshots/screenshot-hdfs-config.png)
+
+Save configurations and restart required services.
+
+# Validate HDFS service
```
Comment (ryanmcdowell) [inline tools/cloudera-parcel-gcsconnector/README.md]: Should be sub-heading

```md
#### Validate HDFS service
```

File: tools/cloudera-parcel-gcsconnector/README.md [hunk 7]
```diff
@@ -0,0 +1,107 @@
...
+$ export JAVA_HOME=/usr/lib/jvm/jre-1.8.0-openjdk.x86_64/
+$ export HADOOP_CLASSPATH=$HADOOP_CLASSPATH:/opt/cloudera/parcels/gcsconnector-1.0.0/lib/hadoop/lib/gcs-connector-latest-hadoop2.jar
+```
+
+Run the 'hdfs dfs -ls' command to access GCS bucket:
+```
+hdfs dfs -ls gs://bucket_name
+```
+
+# Spark Service
```
Comment (ryanmcdowell) [inline tools/cloudera-parcel-gcsconnector/README.md]: Should be sub-heading

```md
### Spark Service
```

File: tools/cloudera-parcel-gcsconnector/README.md [hunk 8]
```diff
@@ -0,0 +1,107 @@
...
+$ spark-shell
+```
+Read file stored on Cloud Storage by providing the gs:// path of a JSON file stored under GCS bucket.
+```
+val src=sqlContext.read.json("gs://bucket-name/some_sample.json")
+```
+
+![alt text](screenshots/screenshot-spark-validate.png)
+
+# Hive Service
```
Comment (ryanmcdowell) [inline tools/cloudera-parcel-gcsconnector/README.md]: Should be sub-heading

```md
### Hive Service
```

Comment (yhqs540) [reply tools/cloudera-parcel-gcsconnector/README.md]: Updates all of the comments

**Pull request merged**